### PR TITLE
Feature/e UI 3750 display role type for assign and reassign

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3750

### Change description ###

- Add a new component for choose-a-role
- Update the roles and access routes 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/work-allocation-2/work-allocation2.module.ts [hunk 1]
```diff
@@ -28,9 +31,11 @@ import { workAllocationRouting } from './work-allocation-feature.routes';
     workAllocationRouting,
     CdkTableModule,
     ExuiCommonLibModule,
-    PriorityFieldComponentModule
+    PriorityFieldComponentModule,
+    ReactiveFormsModule,
+    RoleAccessModule
   ],
-  declarations: [...fromContainers.containers],
+  declarations: [...fromContainers.containers, TaskAssignmentChooseRoleComponent],
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/work-allocation2.module.ts:38]: Why is TaskAssignmentChooseRoleComponent mentioned here?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Few comments to make

File: src/work-allocation-2/containers/task-action/task-action-container.component.spec.ts [hunk 2]
```diff
@@ -202,8 +201,7 @@ describe('WorkAllocation', () => {
               useValue: {
                 snapshot: {
                   data: {
-                    taskAndCaseworkers: {
-                      task: { task: mockTasks[0]}, caseworkers: []},
+                    taskAndCaseworkers: { data: mockTasks[0] },
```
[the remark this answers is not in the document]
Comment (kuda1992) [reply src/work-allocation-2/containers/task-action/task-action-container.component.spec.ts:204]: merge gone wrong

**Review by connorpgpmcelroy — APPROVED** [review]
Comments addressed - LGTM

**Review by udaydenduluri33 — APPROVED** [review]

**Pull request merged**